### ISSUE TITLE
Q: Who should create the Folder $ReadTheDocsPath = Join-Path $BuildDocsPath 'ReadTheDocs'?

### ISSUE THREAD

**Issue by schittli** [zloeber/ModuleBuild#9]
Good morning

**If I `.\Build.ps1` a simple Testproject**, then I get the Exception, that the Folder `C:\Temp\--\-PSModul\jig\build\docs\ReadTheDocs` is missing:
```
Task /./CreateHelp/CreateProjectHelp/UpdateReadTheDocs                                                                
* Copy ReadTheDocs markdown files to project root document folder                                                     
ERROR: Cannot find path 'C:\Temp\--\-PSModul\jig\build\docs\ReadTheDocs' because it does not exist.                   
At C:\Temp\--\-PSModul\jig\jig.build.ps1:659 char:19                                                                  
+ ... $RTDFolders = Get-ChildItem -Path $ReadTheDocsPath -Directory | Sort- ...                                       
+                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                   
```
**If I review `task UpdateReadTheDocs`**, then it except that the Directory `$ReadTheDocsPath` must already exist:
```
# Synopsis: Update ReadTheDocs project documentation
task UpdateReadTheDocs -if {$Script:BuildEnv.OptionGenerateReadTheDocs} {
    Write-Description White 'Copy ReadTheDocs markdown files to project root document folder' -accent

    $BuildDocsPath = Join-Path $BuildRoot "$($Script:BuildEnv.BuildToolFolder)\docs\"
    $ProjectDocsPath = Join-Path $BuildRoot 'docs'
    $ReadTheDocsPath = Join-Path $BuildDocsPath 'ReadTheDocs'
    $DocsReleasePath = Join-Path $Script:BuildEnv.BaseReleaseFolder $Script:BuildEnv.ModuleToBuild

    $RTDFolders = Get-ChildItem -Path $ReadTheDocsPath -Directory | Sort-Object -Property Name
    (…)
}
```
**Therefore my question: who should create this path: `Join-Path $BuildDocsPath 'ReadTheDocs'`?**

**I guess**, it should be either the `Initialize-ModuleBuild` or the `"Docs creation task"`.

Thanks a lot for your help, kind regards, Tom

**Comment by zloeber** [comment]
Good point, fixed in 0.2.3. I'll be pushing up to the gallary shortly as well.

Thanks!

**Comment by schittli** [comment]
Thank you very much for your support!
(I hope this comment does not reopen this issue...)

**Comment by zloeber** [comment]
You are the one supporting the project here, thank you :)

Curious if you have any published modules using ModuleBuild as the framework?

**Comment by schittli** [comment]
Good evening Zachary

I'm very sorry for my late answer... I 
I've just started to create a module containing a collection of our PowerShell-Scripts. I guess most of the 'first modules' start like this :-)

I just want to make sure that later extensions are easy to implement (and all the used tools works as expected), therefore I started to modify the tools around ModuleBuild... I hope this won't be an endless story ;-)

I will ask some questions about my modifications in a separate 'Issue'. If you're interested in some changes, I'll create some Pull Requests as soon I have solved a task.

Kind regards, 
Thomas
